### ISSUE TITLE
BarChart: no tooltip after selecting item from legend with stack option turned on

### ISSUE THREAD

**Issue by GrzegorzLyp** [grafana/grafana#63917]
**What happened**:
Bar Chart with legend and stack option turned on does not show tooltip on bar when you choose not first item in legend.

**What you expected to happen**:
Tooltips should be shown in bar after selecting not first item in legend.

**How to reproduce it (as minimally and precisely as possible)**:
https://play.grafana.org/d/ktMs4D6Mk/5-bar-charts-and-pie-charts?orgId=1&editPanel=15
Change type to stacked one (Normal). Select 2nd item in the legend. Hover the bar on the graph. No tooltip is shown.

![image](https://user-images.githubusercontent.com/29247982/222092330-40158c62-e36c-4158-b3e2-6dab9f16be22.png)

**Environment**:
- Grafana version: 9.4.1 (latest)

**Comment by GrzegorzLyp** [comment]
Will this problem be solved with Grafana 9.5?

**Comment by fheck** [comment]
Can see the same behavior also when hiding a series permanently per override. JavaScript error shows at the same line as with the original steps.

```
Uncaught TypeError: Sa.zZ(...) is undefined                                                    [bars.ts:499:36]
    bbox bars.ts:499
    Tc uPlot.esm.js:4928
    Vf uPlot.esm.js:5160
(webpack://grafana/public/app/plugins/panel/barchart/bars.ts)
    bbox bars.ts:499
    Tc uPlot.esm.js:4928
    Vf uPlot.esm.js:5160
```

Version: Grafana v9.5.2 (cfcea75916)

**Comment by GrzegorzLyp** [comment]
Solved in 10.0.1 
https://github.com/grafana/grafana/pull/69931

**Comment by fheck** [comment]
Can confirm, not seeing this problem any more. Thanks for fixing.
